src/UI/Wanted/Cutoff/CutoffUnmetCollection.js

```diff
@@ -1,18 +1,18 @@
 var _ = require('underscore');
-var EpisodeModel = require('../../Series/EpisodeModel');
+var MovieModel = require('../../Movies/MovieModel');
 var PagableCollection = require('backbone.pageable');
 var AsFilteredCollection = require('../../Mixins/AsFilteredCollection');
 var AsSortedCollection = require('../../Mixins/AsSortedCollection');
 var AsPersistedStateCollection = require('../../Mixins/AsPersistedStateCollection');
 
 var Collection = PagableCollection.extend({
     url       : window.NzbDrone.ApiRoot + '/wanted/cutoff',
-    model     : EpisodeModel,
+    model     : MovieModel,
     tableName : 'wanted.cutoff',
 
     state : {
         pageSize : 15,
-        sortKey  : 'airDateUtc',
+        //sortKey  : 'inCinemas',
```

Comment:
Throws an exception because the cutoffmodule hasn't been updated to suit movies rather than episodes yet and it's beyond my skills to do it at this stage.